public/src/cards.js

```diff
@@ -95,6 +95,9 @@ let events = {
       pack[name] = 1
     }
     App.update()
+    if (App.state.beep){
+      self.beep.play()
+    }
```

Comment:
Hmm. `self` was defined for me, which is why I opted for that route, but I can switch it back. And the braces are remnants of debugging I forgot to remove.